docs/index.rst

```diff
@@ -142,6 +138,10 @@ There are additional tools of use for developers in the
    development/astropy-package-template
    changelog
 
+There are some additional tools of use for developers in the
```

Comment:
Actually the aim of this repo changed a bit over the years, and I think it's more accurate to call it tools for maintainers.

Also, it has been very recently renamed to `astropy-procedures`, please do update it.